website/components/BottomNavigation/index.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import StyledLink from 'components/StyledLink';
+import { Typography } from '@strapi/design-system/Typography';
+import { Flex } from '@strapi/design-system/Flex';
+import Link from 'next/link';
+
+const Wrapper = styled.div`
+  border-top: ${(props) => `1px solid ${props.theme.colors.neutral200}`};
+`;
+
+const BottomNavigation = ({ next, previous }) => {
+  return (
+    <Wrapper>
+      <Flex justifyContent="space-between" paddingTop={8}>
+        <div>
```

Comment:
Would it be possible to replace all the divs in this component with <Box> from the DS?